### DISCUSSION TITLE
3 bugs/problems Electribes 2 sampler

### DISCUSSION THREAD

**Discussion by 34u51** [bangcorrupt/hacktribe#325] [Q&A] [answered]
Hello ....

I have 3 bugs/problems on both Electribes 2 samplers....

when "exporting all samples" they freeze at 1%.... it used to be different with identical firmware...

with "time-slicing" the individual steps are not in their original length and something is missing at the end... Gate is fully open...

Your own samples cannot be gated in “Step mode”.

Hacktribe "update" and SD replacement didn't solve/change it either...

beats and greatz

**Comment by bangcorrupt** [comment]
Do these issues occur only with Hacktribe, or also with the factory firmware?

> when "exporting all samples" they freeze at 1%

If you import [this](https://github.com/bangcorrupt/hacktribe/blob/main/content/carved_beats_set/e2sSample.all) sample set, then export it, does this issue still happen?

> with "time-slicing" the individual steps are not in their original length

I don't know, maybe related to the threshold setting when slicing, but I thought this only effects the onset.

> Your own samples cannot be gated in “Step mode”

I think you need the sample set in loop mode before gating will work.

**Comment by 34u51** [comment]
First of all, a big thank you for making your time and know-how available to us and improving the machine... Hacktribe is super great!!!!

>1 the problem was with firmware & Hacktribe... the export worked once and the only thing that changed was the wav file in the e2Sample.all... I thought I could fix the problem with Hacktribe...

>2 I've now tried several sample sets.... the problem is my e2sSample.all....

>3 I've now read complaints about it on various sites

 >4 Unfortunately looping doesn't help either

(I use a translator if something is expressed incorrectly)

**Reply by 34u51** [reply]
Could this possibly be because I'm rendering wav files [BWF-Chunk] or embedded the tempo???
That's the only thing that's different from previous years....

**Reply by bangcorrupt** [reply, marked as answer]
Extra metadata can cause problems.  Try it without and see what happens.

**Reply by 34u51** [reply]
Can I somehow get the wav files exported from the E2sSample.all.... 500 samples have been collected twice over the years.... 

I now also have the additional problem with an Electribe that when I start the pattern the display no longer shows anything ....

**Reply by bangcorrupt** [reply]
> wav files exported from the E2sSample.all

[Oe2sSLE](https://github.com/JonathanTaquet/Oe2sSLE)

See [this comment](https://github.com/bangcorrupt/hacktribe/discussions/73#discussioncomment-1961200).

> when I start the pattern the display no longer shows anything

Can you post a video?

**Comment by 34u51** [comment]
hello

the problem with exporting was that my samples in e2sSampler.all were rendered with metadata (BPM for example) ....


and the video

https://github.com/bangcorrupt/hacktribe/assets/164012348/1782a8d5-66c8-4315-a750-9228f99119d1

 about the display problem...

**Reply by bangcorrupt** [reply]
This looks like a hardware issue.  I think the power to the display module is getting interrupted, switching off the backlight and resetting the display controller.  The FFC connecting the display is right beneath the play and stop buttons.

Does it happen when you press the stop button?

Does the pattern play, with everything working except the display?

Can you open it up and reseat the FFC in the ZIF connector?

It's the smaller brown cable, it's soldered to the display module at one end and plugs into the motherboard at the other.  You will need to remove the four screws securing the motherboard and carefully fold it over to reach the connectors.  The FFC is quite delicate and could be damaged with a sharp crease or small tear.  If it's not the FFC or connector then it could be a dry solder joint or cracked PCB, or maybe something shorting out if the PCB flexes.
